sandbox/grommet-app/src/components/Metric/Metric.jsx

```diff
@@ -1,39 +1,52 @@
+import React from 'react';
 import { Box, Text } from 'grommet';
 import PropTypes from 'prop-types';
+import { metricSizes } from './sizes';
 
-const sizes = {
-  small: {
-    label: 'small',
-    value: 'xlarge',
-    unit: 'xsmall',
-  },
-  medium: {
-    label: 'medium',
-    value: 'xxlarge',
-    unit: 'small',
-  },
-  large: {
-    label: 'large',
-    value: '3xl',
-    unit: 'medium',
-  },
-};
-export const Metric = ({ label, value, unit, options, size = 'medium' }) => (
-  <Box>
-    <Text size={sizes[size].label}>{label}</Text>
-    <Text size={sizes[size].unit}>
-      <Text size={sizes[size].value} weight={500} color="text-strong">
-        {Intl.NumberFormat(undefined, options).format(value)}
+export const Metric = ({
+  label: labelProp,
+  value: valueProp,
+  unit,
+  options,
+  reverse = false,
+  size = 'medium',
+  ...rest
+}) => {
+  let value = valueProp;
+
+  if (typeof valueProp === 'number') {
+    value = Intl.NumberFormat(undefined, options).format(valueProp);
+  }
+
+  const label = <Text size={metricSizes[size].label}>{labelProp}</Text>;
```

Comment:
Not sure if this is the right place to drop the comment, but something seems like it go messed up on the home page:

BEFORE
<img width="1042" alt="Screenshot 2025-02-20 at 10 48 08 AM" src="https://github.com/user-attachments/assets/8c637952-71a9-4850-b903-a1e8e7a8043c" />

THIS PR
<img width="1068" alt="Screenshot 2025-02-20 at 10 47 49 AM" src="https://github.com/user-attachments/assets/47f05236-1efd-42a4-89ac-431e729a0c1e" />

Reply:
Good catch. I'll track it down.